build-tools/automation/azure-pipelines.yaml

```diff
@@ -691,13 +691,15 @@ stages:
 
     - template: yaml-templates/apk-instrumentation.yaml
       parameters:
-        # TODO: disable LLVM test, see: https://github.com/dotnet/runtime/issues/68914
+        # TODO: disable LLVM test, see:
+        # https://github.com/dotnet/runtime/issues/68914
+        # https://github.com/dotnet/runtime/issues/73304
         condition: false
         configuration: $(XA.Build.Configuration)
         testName: Mono.Android.NET_Tests-AotLlvm
         project: tests/Mono.Android-Tests/Runtime-Microsoft.Android.Sdk/Mono.Android.NET-Tests.csproj
         testResultsFiles: TestResult-Mono.Android.NET_Tests-$(XA.Build.Configuration)AotLlvm.xml
-        extraBuildArgs: -p:TestsFlavor=AotLlvm -p:EnableLlvm=true
+        extraBuildArgs: -p:TestsFlavor=AotLlvm -p:EnableLLVM=true -p:AndroidEnableProfiledAot=false
```

Comment:
Is it necessary to disable profiled AOT?

Reply:
This one was: https://github.com/dotnet/runtime/issues/73003

Using Profiled AOT and LLVM at the same time can hit:

    [ERROR] FATAL UNHANDLED EXCEPTION: System.PlatformNotSupportedException: Arg_PlatformNotSupported
    at System.Text.ASCIIUtility.GetIndexOfFirstNonAsciiByte_Intrinsified(Byte* , UIntPtr )
    at System.Text.Unicode.Utf8Utility.GetPointerToFirstInvalidByte(Byte* , Int32 , Int32& , Int32& )
    at System.Text.UTF8Encoding.GetCharCount(Byte* , Int32 )
    at System.String.CreateStringFromEncoding(Byte* , Int32 , Encoding )
    at System.Text.Encoding.GetString(Byte* , Int32 )
    at System.String.CreateStringForSByteConstructor(Byte* , Int32 )
    at System.String.Ctor(SByte* value)
    at Java.Interop.TypeManager.GetClassName(IntPtr )
    at Android.Runtime.JNIEnv.RegisterJniNatives(IntPtr , Int32 , IntPtr , IntPtr , Int32 )